src/Mutator/MutatorConfig.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Mutator;
+
+class MutatorConfig
+{
+    /**
+     * @var array
+     */
+    private $ignoreConfig;
+
+    public function __construct(array $config)
```

Comment:
how will `array` type declaration work with the following config:

```json
{
    "mutators": {
        "Plus": false
    }
}
```
?

Reply:
Currently it would break on this. However this notation has no use until we implement profiles, which is outside of the scope for this PR.

My plan was to start working on profiles after this, which will involve some refactoring of this class anyways.

Reply:
np. So am I right saying that we won't release 0.9.0 until profiles are implemented?

If so, we can skip creating documentation as well for now

Reply:
Once this gets implemented profiles won't be that big a job, so i'm okay with delaying the doc pr until then.